deps/chakrashim/src/jsrtinspectorhelpers.cc

```diff
@@ -290,15 +335,59 @@ namespace jsrt {
                             &JsConvertValueToString, wasCopied);
   }
 
+  JsErrorCode InspectorHelpers::ArrayConcat(JsValueRef array, JsValueRef value) {
```

Comment:
Honestly, I was trying to avoid writing any JS for this feature since it's hard, if not impossible to debug the shim code that's being used for debugging.  It's much easier to debug the C++ code using jd than to try and figure out what was going on with the shim code.  I can convert some of the simpler ones to shim code, but I like the idea of the majority being in native code to facilitate debugging.